core/xml.js

```diff
@@ -404,8 +400,7 @@ Blockly.Xml.domToWorkspace = function(xml, workspace) {
   }
   var variablesFirst = true;
   try {
-    for (var i = 0; i < childCount; i++) {
-      var xmlChild = xml.childNodes[i];
+    for (var i = 0, xmlChild; xmlChild = xml.childNodes[i]; i++) {
```

Comment:
Isn't this supposed to be `(xmlChild = xml.childNodes[I];)` now?

Reply:
Yeah. Neil took care of that in #3381